app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,47 @@
+<p style="color: green"><%= notice %></p>
+
+
+  <div class="recipe-header">
+      
+    <div class="button-header">
+      <h2 class><%= @recipe.name%></h2>
+    </div>
+
+      <div class="publicR">
+      <h3 class="r-subtitle">
+      Preparation Time: <span class="r-sub-text"><%= @recipe.preparation_time %> min(s)</span>
+      </h3>
+      
+        <% if @recipe.public %>
+        <div class="checker-container">
+          <input class="check-box" type="checkbox" id="checkbox-check" checked />
+          <label class="check-label" for="checkbox-check">Public</label>
+        </div>
+
+        <% end %> <% if !@recipe.public %>
+        <div class="checker-container">
+          <input class="check-box" type="checkbox" id="checkbox-check"/>
+          <label class="check-label" for="checkbox-check">Public</label>
+        </div>
+        <% end %>
+      </div>
+
+    <h3 class="r-subtitle">
+      Cooking Time: <span class="r-sub-text"><%= @recipe.cooking_time %> min(s)</span>
+    </h3>
+
+    <div class="r-sub-text">
+      <h3>Step by step description:</h3>
+      <p class="r-sub-text"><%= @recipe.description %></p>
+    </div>
+
+  </div>
+  
+<div class ="edith-button">
+  <button class="back1"><%= link_to "Edit this recipe", edit_recipe_path(@recipe) %></button>
+  <br>
+  <button class="back1"><%= link_to "Back to recipes", recipes_path %></button>
+<br>
+  <%= button_to "Destroy this recipe", @recipe, method: :delete %>
+</div>
```

Comment:
- [ ] The requirement for the recipe show page is that users should be allowed to add ingredients.
- [ ] On the same recipe show page (when a user clicks a recipe item) should lead to the form that allows the user to add new food, that's if they are the owners of the recipe.
- [ ] The recipe show page should allow users to generate a shopping list. Could you fix all these requirements?

![recipe](https://user-images.githubusercontent.com/53124548/211166776-804a1dc8-e55c-40d8-b004-9192814a384c.png)

Reply:
Thank you for the accurate observations ! The issues have been addressed

Reply:
needs attention!